mak/SRCS

```diff
@@ -75,6 +75,7 @@ SRCS=\
 	src\core\sys\windows\threadaux.d \
 	src\core\sys\windows\windows.d \
 	src\core\sys\windows\winsock2.d \
+	src\core\sys\windows\stdio_msvc12.d \
```

Comment:
ATM you cannot link both stdio_msvc12 and stdio_msvc14 (this is the SRCS file). They both have to be added to MANIFEST, though.